_studio/mfx_lib/genx/CMakeLists.txt

```diff
@@ -77,6 +77,33 @@ if(MFX_ENABLE_KERNELS)
   set( sources "" )
   set( sources.plus "" )
 
+
+  # copy kernels
+  if(BUILD_KERNELS)
+    foreach(platform gen8 gen9 gen10 gen11 gen11lp)
+      add_cmc_target(${MSDK_GENX_ROOT}/copy_kernels/src/genx_copy_kernel.cpp ${platform} ${MSDK_GENX_ROOT}/copy_kernels/isa)
+    endforeach()
+  endif()
+
+  set( prefix ${MSDK_GENX_ROOT}/copy_kernels/isa )
+  list( APPEND sources
+    ${prefix}/genx_copy_kernel_gen8_isa.cpp
+    ${prefix}/genx_copy_kernel_gen9_isa.cpp
+    ${prefix}/genx_copy_kernel_gen10_isa.cpp    
+    ${prefix}/genx_copy_kernel_gen11_isa.cpp
+    ${prefix}/genx_copy_kernel_gen11lp_isa.cpp
+  )
+
+  # field copy
+  set( prefix ${MSDK_GENX_ROOT}/field_copy/isa )
```

Comment:
We don't publish sources for field_copy yet?

Reply:
not yet